Composer/packages/server/src/constants.ts

```diff
@@ -17,4 +17,4 @@ export enum ClaimNames {
 
 export const APPINSIGHTS_INSTRUMENTATIONKEY = process.env.APPINSIGHTS_INSTRUMENTATIONKEY;
 
-export const piiProperties = [];
+export const piiProperties = ['userId', 'rootProjectId', 'projectId'];
```

Comment:
@garypretty Are there any other PII properties that we need to strip out for `TelemetryOptInOut` events? Probably be good to include session id as well.